README.md

```diff
@@ -37,3 +37,14 @@ cd ui
 npm install
 npm start
 ```
+
+# Java Codegen
+
+It is possible to generate java code from the Daml packages you need, by running
+daml with the java codegen option, for example:
+
+```
+daml codegen java -o ./output daml-finance-interface-holding-0.1.7.dar daml-finance-interface-instrument-base-0.1.7.dar daml-finance-interface-instrument-generic-0.1.8.dar daml-finance-interface-lifecycle-0.1.8.dar daml-finance-interface-settlement-0.1.8.dar daml-finance-interface-types-0.1.6.dar daml-finance-interface-util-0.1.6.dar
+```
+
+Note, this only works on packages from Daml Finance release v0.1.7 onwards (November 2022).
```

Comment:
There isn't really a v0.1.7 release of Daml Finance, only individual packages. Maybe just state that it is only supported on SDK versions 2.5.x